values.yaml

```diff
@@ -544,6 +544,13 @@ client:
     #     MIIC7jCCApSgAwIBAgIRAIq2zQEVexqxvtxP6J0bXAwwCgYIKoZIzj0EAwIwgbkx
     #     ...
     caCert: null
+    
+    # Optional fields to control the thresholds of readiness checks.
+    readinessProbe:
```

Comment:
No need to add null fields. you can set the actual values where you want to change otherwise you can put it all in a comment for reference (as documentation)